contents/docs/cdp/batch-exports/bigquery.md

```diff
@@ -0,0 +1,96 @@
+| event                 | `STRING`    | The name of the event that was sent                                                                                 |
+| properties            | `STRING`    | A JSON object with all the properties sent along with an event                                                      |
+| elements              | `STRING`    | A string of elements surrounding an [autocaptured](/docs/data/autocapture) event                                    |
+| set                   | `STRING`    | A JSON object with any person properties sent with the `$set` field                                                 |
+| set_once              | `STRING`    | A JSON object with any person properties sent with the `$set_once` field                                            |
+| distinct_id           | `STRING`    | The `distinct_id` of the user who sent the event                                                                    |
+| team_id               | `STRING`    | The `team_id` for the event                                                                                         |
+| ip                    | `STRING`    | The IP address that was sent with the event                                                                         |
+| site_url              | `STRING`    | The $current_url property of the event. This field has been kept for backwards compatibility and will be deprecated |
+| timestamp             | `TIMESTAMP` | The timestamp associated with an event                                                                              |
+| bq_ingested_timestamp | `TIMESTAMP` | The timestamp when the event was sent to BigQuery                                                                   |
+
+## Creating the batch export
+
+1. Navigate to the exports page in your PostHog instance (Quick links if you use [PostHog Cloud US](https://app.posthog.com/batch_exports) or [PostHog Cloud EU](https://eu.posthog.com/batch_exports)).
+2. Click "Create export workflow".
+3. Select **BigQuery** as the batch export destination.
+4. Fill in the necessary [configuration details](#bigquery-configuration).
+5. Finalize the creation by clicking on "Create".
+6. Done! The batch export will schedule its first run on the start of the next period.
+
+## BigQuery configuration
+
+Configuring a batch export targeting BigQuery requires the following BigQuery-specific configuration values:
+* **Table ID:** The ID of a BigQuery table where to export the data. This is not the fully-qualified name of a table, so omit the dataset and project IDs. For example for the fully-qualified table name `project-123:dataset:MyExportTable` only `MyExporTable` is to be used as the table ID.
+* **Dataset ID:** The ID of a BigQuery dataset where the table to export the data is located. Only the dataset ID is required, so omit the project ID. For example for the dataset `project-123:my-dataset` only `my-dataset` is to be used as the dataset ID.
```

Comment:
Is `exported data table` correct? The wording is a bit confusing here, maybe I got it wrong too. 
```suggestion
* **Dataset ID:** The ID of a BigQuery dataset where the exported data table is located. Only the dataset ID is required, so omit the project ID. For example for the dataset `project-123:my-dataset`, only use `my-dataset` as the dataset ID.

```

Reply:
A google cloud project can contain one or more BigQuery datasets. We are batch exporting data to one table in one of these datasets. So maybe, something more like this?:
> * **Dataset ID:** The ID of the BigQuery dataset which contains the exported data table.

Reply:
That's good

Reply:
I've  tweaked it to say 
> ... which contains the destination table
Just to link it with the previous configuration item (Table ID) as that one is defined as the "... destination BigQuery table".

Hopefully this works!